src/features/homepage/components/latestNews/index.js

```diff
@@ -10,7 +11,7 @@ const LatestNews = ({
   headerImage,
   slug,
 }) => (
-  <Background backgroundImage={headerImage.file.url}>
+  <Background to={handleSlug(slug)} backgroundImage={headerImage.file.url}>
```

Comment:
@codedev-exp the NewsCard component inside is also a Link, its invalid to have a <a> inside an <a>. Styled components should have a polymorphic prop `as` so you can change it to a div here (e.g. <NewsCard as="div">). However, you will also need to make it so the hover style applies to the card text (green underline) when hovering the image too.